tutorials/install-ubuntu-server/install-ubuntu-server.md

```diff
@@ -1,64 +1,164 @@
 ---
 id: tutorial-install-ubuntu-server
-summary: A very high-level guide on how to install Ubuntu Server for general use.
+summary: Turn your PC into a powerful server, capable of delivering anything from file sharing and local backup, to fully fledged web sites and beyond.
 categories: server
 tags: tutorial,installation,ubuntu,server,Ubuntu 16.04 LTS
 difficulty: 2
 status: Published
-published: 2017-05-31
+published: 2017-07-07
 
 ---
 
-# Installing Ubuntu Server for general use
+# Install Ubuntu Server
 
-## Getting started
-Duration: 0:05
+## Overview
+Duration: 0:01
 
-The basic steps to install Ubuntu Server from CD or USB stick are the same for installing any operating system. Unlike the desktop version, Ubuntu Server does not include a graphical installation program. Instead, it uses a console menu-based process.
+Ubuntu server is Ubuntu tailored for networks and services, and it's just as capable sharing files on your home network as it is running a 50,000 render farm.
 
-positive
-: Note - this is a very high-level guide. For a more detailed guide go to the [Ubuntu Server help documentation](https://help.ubuntu.com/lts/serverguide/installation.html
+Unlike the desktop version of Ubuntu, Ubuntu Server does not include a graphical installation program. Instead, it uses a console menu-based process. If you'd rather install the desktop version, take a look at our [Install Ubuntu desktop](https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-deskto) tutorial.
+
+This guide will be a brief overview of the installation from either a DVD or a USB flash drive, suitable for anyone with an interest in running their own server. For a more detailed guide on Ubuntu Server's capabilities and its configuration, take a look at our [Ubuntu Server help documentation](https://help.ubuntu.com/lts/serverguide/installation.html
 ).
 
-## Prepare a DVD or USB flash drive
-Duration: 0:10
+![screenshot](https://assets.ubuntu.com/v1/fcc62c28-server-welcome.png)
+
+## Requirements
+Duration: 0:01
+
+You'll need to consider the following before starting the installation:
+
+* Ensure you have at least 2GB of free storage space. 
+* Have access to either a DVD or a USB flash drive containing the version of Ubuntu Server you want
+  to install.
+* Make sure you have a recent backup of your data. While it's unlikely that
```

Comment:
this is all for folks new to installing Ubuntu server. It's kind of a caveat. I'll add some clarification.